### ISSUE TITLE
[CHANGE]

### ISSUE THREAD

**Issue by nathangilchrist** [rohanb10/snoozz-tab-snoozing#92]
## What would you like to change

The order of the buttons that shift the intended snooze time by set increments is in the opposite order from the function provided by the Increase/Decrease buttons inset in the numeric fields. While up increases the number by one is on top, the buttons below that are quick-shifts has the increasing option on the bottom.

## What would you like to see instead

Swap the order of the incremental time shifters. For instance, "-5 minutes" should come below "+5 minutes"

## Screenshots / Videos

![image](https://user-images.githubusercontent.com/7089117/152633839-ec2556c9-d365-4187-9c48-fd2d750b21a0.png)

**Comment by rohanb10** [comment]
Yeah makes sense. Fixed in https://github.com/rohanb10/snoozz-tab-snoozing/commit/5d2c971fdb300e521b115f9b310641fcc4c16ead

Will be pushed with the next release
